controllers/subscriber/subscriberControllers.js

```diff
@@ -118,9 +121,7 @@ export class IdSubscriberController {
 		});
 	}
 
-	hostUpdated(changedProperties) {
```

Comment:
The hostUpdated lifecycle method has no parameters, so changedProperties will always be undefined. Removing the check allows the tests to pass properly. I'm not too sure on the effects this might have to the below code though - haven't done too much digging.

Context: https://lit.dev/docs/api/controllers/#ReactiveController.hostUpdated

Reply:
Oh interesting!  This makes their `hostUpdated` function much less useful lol.  If we wanted to avoid the consumer having to know to call this, I guess we could keep track of the previous value and do our own comparison each time `updated` is called, but that feels like a waste when Lit already does exactly that comparison to create `changedProperties`... 🤔 

So I'm good with this strategy unless others feel strongly about hiding this from the consumer!

Reply:
I like that idea Stacey. I've pushed it here, but I can revert if we like the other approach better. We know that the property is a space-delimited string, so the comparison each time shouldn't be too performance-intensive.

Reply:
Right good call, I was still thinking the property was an array (it was initially) and would be complicated to compare.  Nice!